starknet_py/net/http_client.py

```diff
@@ -85,7 +85,7 @@ async def handle_request_error(self, request: ClientResponse):
 
 
 class RpcHttpClient(HttpClient):
-    async def call(self, method_name: str, params: dict) -> dict:
```

Comment:
Was it needed because of the `bool` in `syncing` method?

Reply:
The typing was wrong, the result is not always a dict, some methods return just a number (chain id), syncing just a bool, etc.